### DISCUSSION TITLE
请问terrian的风向图的最后的两维是风速的x，y分量吗，还是角度和大小？

### DISCUSSION THREAD

**Discussion by Lodfy-Feng** [physics-data/faq#71] [Q&A] [answered]
此外助教给的example是正确的吗？其他同学按照反向给出的图似乎更正常。
![image](https://user-images.githubusercontent.com/63779478/130937038-4b26a890-37d3-4989-b225-9b3e6cb79962.png)

**Comment by jiegec** [comment, marked as answer]
请查看网络学堂公告
